tools/topology/topology2/cavs-rt5682-ssp0spk.conf

```diff
@@ -0,0 +1,291 @@
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<pdm_config.conf>
+<tokens.conf>
+<virtual.conf>
```

Comment:
do you really need virtual.conf? or is it a cut, copy & paste issue?

Reply:
it is a copy paste issue Ranjani...will check that